src/styles/utils/variables.scss

```
@@ -0,0 +1,5 @@
+$color-blue-accent: #00acdc;
+$color-main-accent: #060b35;
+$color-gray: #616070;
+$transition-duration: 0.3s;
+$card-width: 200px;
```

It is better to use more universal naming for variables.

Example:

```suggestion
$accent-color: #00acdc;
$main-color: #060b35;
$second-color: #616070;
$transition-duration: 0.3s;
$card-width: 200px;
```